packages/blaze/template.js

```diff
@@ -318,6 +322,23 @@ Template.prototype.helpers = function (dict) {
     this.__helpers.set(k, dict[k]);
 };
 
+// Kind of like Blaze.currentView but for the template instance.
+// This is a function, not a value -- so that not all helpers
+// are implicitly dependent on the current template instance.
```

Comment:
That's not exactly right, because the return value of the template instance function is constant, and we don't do any Tracker signaling on it as far as I know.

What's actually going on with the template instance is it has a bunch of fields that are updated "on demand," when someone looks at it.

Another way to implement this would be to store either the view or the template instance, and from `Template.instance()` call `Template._currentTemplateInstance.view.templateInstance()`, or `Template._currentTemplateView.templateInstance()`.